### DISCUSSION TITLE
Scroll content view to top when bottomSheetPosition in middle

### DISCUSSION THREAD

**Discussion by thangldvn** [LucasMucGH/BottomSheet#93] [Q&A] [answered]
How can i scroll main content view to top when bottomSheetPosition is in middle

**Comment by lucaszischka** [comment, marked as answer]
Do you have `.appleScrollBehaviour` enabled? If so, you can’t scroll at all when in the middle position.

**Reply by thangldvn** [reply]
yes. i have enabled . appleScrollBehaviour. but when i scroll content to specific position instead of top position and i dragged indicator to middle, the content not reset to top position

**Reply by lucaszischka** [reply]
Yes it doesn’t? This how Apple BottomSheet behaves as well in Apple Maps for example. So this is the expected behavior.
